### DISCUSSION TITLE
GPIO problem

### DISCUSSION THREAD

**Discussion by laguinch** [photobooth-app/photobooth-app#192] [Q&A]
Hy, if I take a photo with the gpio pin (pin 27 single photo) the Photo Booth take more photo it's always a different number of photos) and I have the 9 dots between all countdown. I'm sure I'm not in the collage configuration. I have the same problem with the collage button it's take a lot photos and I have 3 or more collage picture in the gallery and a least I have the message error "oh no stream error"
With the button on the screen I have no problem
Thx

**Comment by mgineer85** [comment]
Hey, what kind of button you have connected?
Maybe the button triggers multiple times? Pls post the logs here.

**Reply by laguinch** [reply]
The button work perfectly a few month a go. I try a new fresh install. I'll give you the feedback

**Reply by laguinch** [reply]
it's the same with the fresh install. See the log in the file Thx
I test with only to wire with a lite touch and it's the same
[latest.txt](https://github.com/photobooth-app/photobooth-app/files/14731773/latest.txt)

**Reply by mgineer85** [reply]
Hi, from the logs I see that the gpio triggers servers times right after each other. 
could you test somehow that the button is actually behaving correctly and not faulty? To the app it looks like it’s pressed several times…
maybe something in the underlying software changed recently?
How exactly is the button wired?

**Reply by mgineer85** [reply]
When it was working correctly, on which version have you been? 
It started failing after an app update?

**Reply by laguinch** [reply]
Hy, the button is a simple push button  like this one https://amzn.eu/d/hGMQG3T it’s wired directly on the GPIO pin. 
I test with just 2 wire with a lite touch and it’s the same. 
the last Time it was working in November 2023 the 1.9 version
